python_bindings/bindings.cpp

```diff
@@ -0,0 +1,659 @@
+#include "agent.hpp"
+#include "agent_io.hpp"
+#include "agents/activity_agent.hpp"
+#include "agents/discrete_vector_agent.hpp"
+#include "agents/inertial_agent.hpp"
+#include "agents/simple_agent.hpp"
+#include "config_parser.hpp"
+#include "model.hpp"
+#include "model_factory.hpp"
+#include "models/ActivityDrivenModel.hpp"
+#include "models/DeGroot.hpp"
+#include "models/DeffuantModel.hpp"
+#include "models/InertialModel.hpp"
```

Comment:
See for example https://github.com/flowy-code/flowy/commit/5ef73d68ef83946359295c7259f798de32080ea3